### PR TITLE
2022 05 26

### PR DESCRIPTION
**Pull request by odscjames**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 
 jobs:
-  build:
+  test:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [3.8, 3.9, "3.10"]
+        django-version: [">=3.2,<4", ">=4,<5"]
```
Comment (Bjwebb) [inline .github/workflows/test.yml:10]: I would also test against 3.2.x, as it's the most recent LTS.

**Review by Bjwebb — APPROVED** [review]
I've added a comment about Django versions for testing, and only skimmed the example changes, but generally looks good.

File: .github/workflows/test.yml (hunk 1 above)
Comment (odscjames) [reply .github/workflows/test.yml:10]: You mean Django? 

It does - You can see in https://github.com/OpenDataServices/json-data-ferret/runs/6608907132?check_suite_focus=true - open the "run pip install Django" step and you can see it installs Django 3.2.13.

https://github.com/OpenDataServices/json-data-ferret/runs/6608907261?check_suite_focus=true installs 4.0.4, so we are covering both 3.2 & 4.

I was careful to get both - Golab is currently on 3.2 and I don't want to be forced to upgrade that right now but we will have to at some point and I wanted to make sure we were ready.

**Comment by odscjames** [comment]
I think I've already covered your point so I merged but if you can show I messed that up I will definitely look into that again as I totally agree.

File: .github/workflows/test.yml (hunk 1 above)
Comment (Bjwebb) [reply .github/workflows/test.yml:10]: Ah yes, this is fine. I was thinking there was a 3.3, but there isn't.

**Pull request merged**